scram-tools.file/tools/rocm/rocm.xml

```diff
@@ -0,0 +1,36 @@
+<tool name="rocm" version="@TOOL_VERSION@">
+  <info url="https://docs.amd.com/"/>
+  <lib name="amdhip64"/>
+  <client>
+    <environment name="ROCM_BASE" default="@TOOL_ROOT@"/>
+    <environment name="ROCM_LLVM" default="$ROCM_BASE/llvm/lib/clang/14.0.0"/>
+    <environment name="HIPCC"     default="$ROCM_BASE/bin/hipcc"/>
+    <environment name="BINDIR"    default="$ROCM_BASE/bin"/>
+    <environment name="LIBDIR"    default="$ROCM_BASE/lib"/>
+    <environment name="LIBDIR"    default="$ROCM_BASE/lib64"/>
+    <environment name="LIBDIR"    default="$ROCM_BASE/hip/lib"/>
+    <environment name="LIBDIR"    default="$ROCM_BASE/hsa/lib"/>
+    <environment name="LIBDIR"    default="$ROCM_LLVM/lib/linux"/>
+    <environment name="LIBDIR"    default="$ROCM_BASE/llvm/lib"/>
+    <environment name="INCLUDE"   default="$ROCM_BASE/include"/>
+    <environment name="INCLUDE"   default="$ROCM_BASE/hip/include"/>
+    <environment name="INCLUDE"   default="$ROCM_BASE/hsa/include"/>
+    <environment name="INCLUDE"   default="$ROCM_LLVM"/>
+  </client>
+  <flags ROCM_FLAGS="-fno-gpu-rdc --amdgpu-target=gfx900 --gcc-toolchain=$COMPILER_PATH -D__HIP_PLATFORM_HCC__ -D__HIP_PLATFORM_AMD__"/>
+  <!-- REM_CXXFLAGS from llvm/llvm-cxxcompiler.xml -->
+  <flags ROCM_HOST_REM_CXXFLAGS="-Wno-non-template-friend"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-Werror=format-contains-nul"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-Werror=maybe-uninitialized"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-Werror=unused-but-set-variable"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-Werror=return-local-addr"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-fipa-pta"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-frounding-math"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-mrecip"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-fno-crossjumping"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-fno-aggressive-loop-optimizations"/>
+  <flags ROCM_HOST_REM_CXXFLAGS="-funroll-all-loops"/>
+  <flags ROCM_HOST_CXXFLAGS="-D__HIP_PLATFORM_HCC__ -D__HIP_PLATFORM_AMD__"/>
+  <runtime name="ROOT_INCLUDE_PATH" value="$INCLUDE" type="path"/>
```

Comment:
@fwyzard , `ROOT_INCLUDE_PATH` is only needed if `rocm` headers are used for root dictionaries. I think we do not want to do this, so better to remove this line.

Reply:
Currently we have it for CUDA:
https://github.com/cms-sw/cmsdist/blob/76762e985cc3667721e19c1b3795e4d7e8d8630d/scram-tools.file/tools/cuda/cuda.xml#L19

I'd like to keep it also for ROCm, until we figure out how to handle the (transient) data formats and their dictionaries.